src/Runtime/python/onnxmlir/tests/use_compiler_container.py

```diff
@@ -0,0 +1,21 @@
+# Test: no name for input and output
+import numpy as np
+import onnxmlir
+
+a = np.arange(3 * 4 * 5, dtype=np.float32).reshape((3, 4, 5))
+b = a + 4
+
+sess = onnxmlir.InferenceSession(
+    "test_add.onnx",
+    compile_args="-O3 --parallel",
+    container_engine="docker",
```

Comment:
Docker vs Podman. If there was a way to "query" which package was installed (as we had to do a `install onnxmlir[docker]` vs `onnxmlir[podman]`, that would be one less option that seems redundant.